azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -599,21 +604,20 @@ foreach ($cluster in $AgConfig.SiteConfig.GetEnumerator()) {
 }
 
 #####################################################################
-# Setup Azure Container registry on AKS Edge Essentials clusters
+# Setup Azure Container registry pull secret on clusters
 #####################################################################
-Write-Host "[$(Get-Date -Format t)] INFO: Configuring secrets on clusters (Step 9/12)" -ForegroundColor DarkGreen
+Write-Host "[$(Get-Date -Format t)] INFO: Configuring secrets on clusters (Step 9/13)" -ForegroundColor DarkGreen
 foreach ($cluster in $AgConfig.SiteConfig.GetEnumerator()) {
-    if ($cluster.Value.Type -eq "AKSEE") {
-        Write-Host "[$(Get-Date -Format t)] INFO: Configuring Azure Container registry on ${cluster.Name}"
-        kubectx $cluster.Name.ToLower() | Out-File -Append -FilePath ($AgConfig.AgDirectories["AgLogsDir"] + "\ClusterSecrets.log")
+        $clusterName = $cluster.Name.ToLower()
+        Write-Host "[$(Get-Date -Format t)] INFO: Configuring Azure Container registry on $clusterName"
+        kubectx $clusterName | Out-File -Append -FilePath ($AgConfig.AgDirectories["AgLogsDir"] + "\ClusterSecrets.log")
```

Comment:
@dkirby-ms  - 
While on the subject of Functions (from my comment below) the repeated use of Out-File would be another good place.

Function call could be something like:
`| Out-Log -StepName $stepName`
Where $stepName is "ClusterSecrets" in this case

Then the function can do the:
` | Out-File -Append -FilePath ($AgConfig.AgDirectories["AgLogsDir"] + "\$stepName.log")`

As a nice bonus, the function can also add the $currentStep value (again from comment below) so that the log files in the logs folder line up in step order, not alpha order:
```
...
Step7_foo.log
Step8_ClusterSecrets.log
Step9_blah.log
...
```
This would create multiple log files per Step in some cases, but it will create a clear line from log file back to code.